api/internal/infrastructure/mongo/mongodoc/project.go

```diff
@@ -26,12 +26,9 @@ type ProjectDocument struct {
 	PublicImage       string
 	PublicNoIndex     bool
 	Workspace         string // DON'T CHANGE NAME'
-	Visualizer        string
-	PublishmentStatus string
-	CoreSupport       bool
+	Workflow          string
```

Comment:
Is this meant for WorkflowID? If so, please change the name to workflowId instead of workflow